trigger-workflow/index.js

```diff
@@ -0,0 +1,152 @@
+/**
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+const core = require('@actions/core');
+const github = require('@actions/github');
+
+let octokit;
+
+function sleep(milliseconds) {
+  return new Promise((r) => setTimeout(r, milliseconds));
+}
+
+/**
+ * Get the workflow runs for the provided workflow id.
+ */
+async function getWorkflowRuns(owner, repo, workflowID) {
+  const { data: runs } = await octokit.rest.actions.listWorkflowRuns({
+    owner,
+    repo,
+    workflow_id: workflowID,
+  });
+  return runs.workflow_runs;
+}
+
+/**
+ * Get the run that was triggered using the dispatch event.
+ */
+async function getWorkflowRun(owner, repo, workflowID, srcWfRunID) {
+  await sleep(10000);
```

Comment:
Is there a possibility that all of these sleeps would not be enough? i.e., that it would take longer for the job to run so that we could find `srcWfRunID`?

Reply:
`scrWfRunID` is defined by the action [here](https://github.com/nuxeo/ui-team-gh-actions/pull/2/files/d82b2a67d39116424d4453a2dfbd08b57a1f8ef9..32fc52c725a1f1c751f6a2d349d820630af956da#diff-f3ffbd1e03efcc3a51c248ff1f4b2567c1095cf0f06711bb7ce3a57034029511R116). and in fact, there's the possibility that the sleeps are not enough to find the run. 

The problem is that there's no proper way for us to know which one is the run we triggered. Think of this as an heuristic to search and get the run. To make this a bit more robust, maybe we could do retries for [this](https://github.com/nuxeo/ui-team-gh-actions/pull/2/files/d82b2a67d39116424d4453a2dfbd08b57a1f8ef9..32fc52c725a1f1c751f6a2d349d820630af956da#diff-f3ffbd1e03efcc3a51c248ff1f4b2567c1095cf0f06711bb7ce3a57034029511R46) and combine it with the sleep, but other than that, I think we shouldn't be polling the called repository indefinitely.